EWS O365 - fix fetch emails (#33444)
demisto/content: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_44.md

#### Integrations

##### EWS O365

> - Fixed an issue where some emails with attachments was not being fetched correctly.

Reviewer: ```suggestion
Fixed an issue where some emails with attachments were not fetched correctly.
```
